lib/solr/params.rb

```diff
@@ -74,6 +73,7 @@ def sort(field, direction)
       raise ArgumentError, "direction must be one of #{VALID_SORT_DIRECTION.join(', ')}, instead got '#{direction}'" unless VALID_SORT_DIRECTION.include?(direction)
 
       @parameters[:sort] = "#{field} #{direction}"
+      self
```

Comment:
Although this comment is out of scope for this PR, it may also be good to return `self` in the above `raw_parameter` method.

Reply:
Good call - added it in a few places to make sure chaining works.

Reply:
Thanks!